instrumentation/grpc/test/support/proto/ping_pb.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
```

Comment:
Is there documentation to regenerate these files? When might someone need to regenerate them?

Reply:
I've added a brief note in the README under the Development section explaining how to regenerate these files  using `grpc_tools_ruby_protoc` https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1303/files#diff-8ec47973e98f69236d5b7f178b429d4d18133f13c80f07a1f7d37042c1e7a11eR51

Do you think this provides enough detail? Is there any specific information you feel should be included?

Reply:
Ah, my mistake! That seems sufficient. Thank you!